TreeMaker/python/makeTreeFromMiniAOD_cff.py

```diff
@@ -1163,6 +1163,100 @@ def makeTreeFromMiniAOD(self,process):
             'trackFilter:trkshitpattern(Tracks_hitPattern)',
         ])
 
+
+    ## ----------------------------------------------------------------------------------------------
+    ## AK15 jets
+    ## ----------------------------------------------------------------------------------------------
+
+    if self.boostedsemivisible:
+
+        from JMEAnalysis.JetToolbox.jetToolbox_cff import jetToolbox
+        jetToolbox(process,
+            'ak15',
+            'jetSequence',
+            'out',
+            PUMethod = 'Puppi',
+            miniAOD = True,
+            runOnMC = self.geninfo,
+            Cut = 'pt>20.',
+            addPruning = True,
```

Comment:
are pruned variables actually used? if not, this could be disabled

Reply:
I don't think so, I've disabled them for now.